docs/gettingStarted/AriesDeveloperDemos.md

```diff
@@ -21,3 +21,12 @@ and then use your new wallet to get and present credentials in some sample scena
 verifiable credential experience in 30 minutes or less.
 
 [BC Gov Showcase]: https://digital.gov.bc.ca/digital-trust/showcase/
+
+## Indicio Developer Demo
+
+Minimal Aca-Py demo that can be used by developers to isolat and test features:
```

Comment:
Small spelling mistake on `islolat`

Reply:
Good catch we can fix in the next PR tho ...